ghost/core/test/e2e-api/admin/pages-bulk.test.js

```diff
@@ -144,10 +144,12 @@ describe('Pages Bulk API', function () {
                             visibility: 'tiers',
                             tiers: [
                                 {
-                                    id: tier1.id
+                                    id: freeTier.id,
+                                    type: freeTier.get('type')
```

Comment:
I don't think we should add this to the API in order to make it work

Reply:
In Admin, when the user right-clicks on a Post and then clicks on "Change Access", we receive complete `tier` objects (so including `id`, and `type`). Our tests didn't reflect that.

Is there any other use case for the bulk edit today, where we would only receive the tier `id`?

Reply:
Not for the bulk API, because I think this one is undocumented, however I would think it is better that it behaves the same as the single edit API. For the 'single edit API', AFAIK your changes also require the type to be set when changing tiers, maybe that isn't covered in tests at the moment, but this should be covered. 

For the single edit API we cannot introduce a breaking change in a not major Ghost version. It is possible that a site is using the admin API directly and is only sending the id's, not the full objects.

Reply:
Fair point — the change in the PR doesn't change the single edit API, but I follow the argument of not making these two differ in requirements. Updated